tslint.json

```diff
@@ -1,82 +0,0 @@
-{
```

Comment:
Could you add this back ? so the tslint scripts work well ?

Reply:
done